README.md

```diff
@@ -2566,6 +2571,34 @@ Nat. Nanotechnol., 2023.
 IEEE Trans. Pattern Anal. Machine Intell. (TPAMI), 2023. [Project page](https://github.com/tub-rip/event_based_bos), [Video](https://youtu.be/Ev52n8KgxIU), [PDF](https://arxiv.org/pdf/2311.00434)
 
 
+
+<a name="driver_monitoring_system"></a>
+## Driver Monitoring System
+<a name="DMS"></a>
+### Face, Head-Pose and Eye-gaze Estimation
+- <a name="Ryan23IEEEAccess"></a>Ryan, Cian and Elrasad, Amr and Shariff, Waseem and Lemley, Joe and Kielty, Paul and Hurney, Patrick and Corcoran, Peter.,  
+*[Real-Time Multi-Task Facial Analytics With Event Cameras](https://openaccess.thecvf.com/content/CVPR2022/html/Lin_Autofocus_for_Event_Cameras_CVPR_2022_paper.html](https://ieeexplore.ieee.org/abstract/document/10188686))*,  
+ IEEE Access, vol. 11, pp. 76964-76976, 2023, doi: 10.1109/ACCESS.2023.3297500.
```

Comment:
We do not write the doi explicitly in the citation. We use it in the link to the official page of the paper.
Please see the convention used in other 10 IEEE Access paper entries currently on the repo.